src/plugins/tables/tables.js

```diff
@@ -135,31 +135,42 @@ $document.on( "init.dt draw.dt", selector, function( event, settings ) {
 		ol = document.createElement( "OL" ),
 		li = document.createElement( "LI" );
 
-	// Update Pagination List
-	for ( var i = 0; i < paginate_buttons.length; i++ ) {
-		var item = li.cloneNode( true );
-		item.appendChild( paginate_buttons[ i ] );
-		ol.appendChild( item );
-	}
+	// Determine if Pagination required
+	if ( paginate_buttons.length < 4 ) {
+		$( ".dataTables_paginate" ).addClass( "hidden" );
+		$( ".dataTables_paginate" ).attr( "aria-hidden", "true" );
```

Comment:
Using the attribute ```aria-hidden``` is not needed because the CSS is an Accessibility-Supported Web Technology

Reply:
Okay that can easily be removed.